lib/helpers/remote_config_helper.dart

```diff
@@ -1,9 +1,32 @@
 import 'package:firebase_remote_config/firebase_remote_config.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:givt_app_kids/features/auth/cubit/auth_cubit.dart';
+import 'package:givt_app_kids/features/flows/cubit/flows_cubit.dart';
+import 'package:givt_app_kids/features/profiles/cubit/profiles_cubit.dart';
+import 'package:givt_app_kids/helpers/analytics_helper.dart';
 
 class RemoteConfigHelper {
   static bool isFeatureEnabled(RemoteConfigFeatures feature) {
     return FirebaseRemoteConfig.instance.getBool(feature.value);
   }
+
+  static bool logoutHelper(BuildContext context) {
```

Comment:
a bit strange method name :)
also, I do not think RemoteConfigHelper is a proper place for that logic..